### ISSUE TITLE
Using with Angular 11

### ISSUE THREAD

**Issue by hioymaci** [flowjs/ngx-flow#45]
I know that ngx-flow is used Angular 8 and below. But how can I use it with v11. Is there any quick way or all library should be implemented for v11?

**Comment by MartinNuc** [comment]
It's compatible with Angular 8+. I am not aware of any issues with newer versions of Angular. Have you tried it with Angular 11?

**Comment by hioymaci** [comment]
I tried with Angular 11 but it doesn't run. Actually, I used to ngx-flow Angular v7. Then I updated my project to v9. There is no problem with ngx-flow. But after I want to update v10 "ng update" doesn't permit. I forced to 11 then I upload mechanism crashed:  something like that: flowjs object doesn't have flow field. Sorry I don't have a lot of experience with Angular.

**Comment by MartinNuc** [comment]
I will have to try it out then.

**Comment by hioymaci** [comment]
> I will have to try it out then.

Thank you Mr. Martin. After your answer I tried again to update angular but this time I updated step by step.
Ngx-flow runs properly with

- Angular v7,
- Angular v8,
- Angular v9,
- Angular v10,

in my application.

I do not know where was the problem. But after I updated from v7 to v11 it crashed. So I updated v8 then v9 then v10. 

v10 is also enough for me. Thank you so much again.
